site/content/docs/user/local-registry.md

```diff
@@ -25,7 +25,7 @@ The registry can be used like this.
 1. First we'll pull an image `docker pull gcr.io/google-samples/hello-app:1.0`
 2. Then we'll tag the image to use the local registry `docker tag gcr.io/google-samples/hello-app:1.0 localhost:5001/hello-app:1.0`
 3. Then we'll push it to the registry `docker push localhost:5001/hello-app:1.0`
-4. And now we can use the image `kubectl create deployment hello-server --image=localhost:5001/hello-app:1.0`
+4. And now we can use the image `kubectl create deployment hello-server --image=kind-registry:5000/hello-app:1.0`
```

Comment:
this change is unrelated and localhost:5001 should work.

the goal is to ensure that a consistent name is used for the images across hosts to avoid confusion for users.